src/regions/ca/info/faq/43_wash-hands-home.md.fr

```diff
@@ -0,0 +1,12 @@
+## Est-ce que si je reste chez moi je dois aussi me laver les mains régulièrement?
+
+Si vous êtes chez vous, il faut vous laver les mains :
+
+- Après avoir toussé, éternué, vous êtes mouché ou avoir touché l’intérieur de votre nez ou votre bouche.
+- Après être allé aux toilettes
+- Avant de préparer des aliments
+- Avec du savon et de l’eau de préférence, pendant plus de 20 secondes, ou avec un désinfectant à base d’alcool pour les mains si vous n’y avez pas accès. Notez que le désinfectant à base d’alcool pour les mains n’est pas efficace si elles sont souillées
+
+Ceci permet la prévention de tout type de contamination.
+
+Plus d'informations disponibles [ici](https://github.com/dialoguemd/covid-19/blob/master/src/regions/ca/info/faq/01_sick-notes.fr.md).
```

Comment:
This links to the GH repo, maybe the link to the FAQ page would be better?

Reply:
Fixed!